src/main/java/de/zalando/aruha/nakadi/validation/EventValidation.java

```diff
@@ -0,0 +1,73 @@
+package de.zalando.aruha.nakadi.validation;
+
+import java.util.List;
+import java.util.Map;
+
+import org.json.JSONObject;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.domain.ValidationStrategyConfiguration;
+
+public class EventValidation {
+
+    private static final Map<String, EventTypeValidator> eventTypeValidators = Maps.newConcurrentMap();
+
+    public static EventTypeValidator forType(final EventType eventType) {
+        return forType(eventType, false);
+    }
+
+    public static EventTypeValidator forType(final EventType eventType, final boolean incrementalEdit) {
```

Comment:
I don't really understand what this method does, especially the `incrementalEdit` parameter